src/test/Testcafe/dashboard_tests/app-tests.js

```diff
@@ -6,40 +6,43 @@ import config from '../config.json';
 
 let app = new App();
 let dash = new DashBoard();
-let environment = (process.env.ENV).toLowerCase();
+let environment = process.env.ENV.toLowerCase();
 
 fixture('Dashboard tests')
   .page(app.baseUrl)
-  .beforeEach(async t => {    
-    t.ctx.existingApp = 'autotestdeploy';    
+  .beforeEach(async (t) => {
+    t.ctx.existingApp = 'autotestdeploy';
     t.ctx.ingenSkriveApper = 'Vi fant ingen apper som du har skriverettigheter til';
     t.ctx.ingenLeseApper = 'Vi fant ingen apper som du eksplisitt har fått leserettigheter til';
-    await t
-      .maximizeWindow()
-      .useRole(AutoTestUser);
+    await t.maximizeWindow().useRole(AutoTestUser);
   });
 
 test('Cannot create new app, as app name already exists', async () => {
   await t
     .click(dash.newAppButton)
     .click(dash.appOwner)
-    .expect(dash.appOwnerList.withExactText('Testdepartementet').exists).ok()
+    .expect(dash.appOwnerList.withExactText('Testdepartementet').exists)
+    .ok()
     .click(dash.appOwnerList.withExactText('Testdepartementet'))
     .click(dash.appName)
     .typeText(dash.appName, t.ctx.existingApp)
     .click(dash.createAppButton)
-    .expect(dash.appExistsDialogue.exists).ok();
+    .expect(dash.appExistsDialogue.exists)
+    .ok();
 });
 
 test('Error messages when app does not exist', async () => {
   var appName = config[environment].deployApp.toString();
   appName = appName.split('/');
   appName = appName[1];
   await t
-    .expect(dash.appsList.child('div').count).gte(1, 'Apps not loaded', { timeout: 60000 }) //To wait until the apps are loaded
+    .expect(dash.appsList.child('div').count)
+    .gte(1, 'Apps not loaded', { timeout: 60000 }) //To wait until the apps are loaded
```

Comment:
Nice!
As I mentioned in the other PR, this is also an option when breaking up into more lines for easier debugging.

Reply:
in case of frontend tests, chaining the assertions is the optimal solution as breaking down in multiple lines, will increase the runtime of the tests. (since the script will again search for the element in DOM to perform the assertion) and in case of debugging, the tool testcafe reports where the test has failed.